docs/packages/pkg/corrade.rst

```diff
@@ -0,0 +1,42 @@
+.. spelling::
+
+    corrade
+
+.. index::
+  single: graphics ; corrade
+
+.. _pkg.corrade:
+
+corrade
+===
+
+-  `Official <https://github.com/mosra/corrade>`__
+-  `Example <https://github.com/ruslo/hunter/blob/master/examples/corrade/CMakeLists.txt>`__
+-  Developed by `Vladimír Vondruš<https://github.com/mosra>`
+-  Added by `Pascal Thomet <https://github.com/pthom>`__ (`pr-1646 <https://github.com/ruslo/hunter/pull/1646>`__)
+
+usage
+-----
+
+.. code-block:: cmake
+
+    hunter_add_package(corrade)
+    find_package(corrade CONFIG REQUIRED TestSuite Utility PluginManager Containers)
+    target_link_libraries(...
+      Corrade::Containers Corrade::Interconnect Corrade::PluginManager
+      Corrade::TestSuite Corrade::Utility
+      )
+
+About
+-----
+
+corrade is a C++11/C++14 multiplatform utility library, which is needed by `magnum<https://github.com/mosra/magnum>`.
+
+magnum is a lightweight and modular C++11/C++14 graphics middleware for games and data visualization
+
+
+Known issues
+------------
+
+Cross compilation to iOS and Android might fail since the build require to find `corrade-rc` (native executable)
```

Comment:
Please change to (*.rst format, not *.md)
```
``corrade-rc``
```

Reply:
Fixed by 432b6f4505ad83062a765c472ceef2ecfeb3ced0 (my bad, I'm not proficient with reStructured Text)